src/root/rmem.d

```diff
@@ -127,7 +127,7 @@ else
     __gshared size_t heapleft = 0;
     __gshared void* heapp;
 
-    extern (C++) void* allocmemory(size_t m_size)
+    extern (C) void* allocmemory(size_t m_size)
```

Comment:
> Not another name debate, sigh.

What are you talking about?  Instead of changing the linkage, do what you did for every other mangling problem in this PR.